### PR TITLE
DDP-8135/osteo welcome button fix

### PR DESCRIPTION
**Pull request by grigol-brd**

### CODE REVIEW

File: ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html [hunk 1]
```diff
@@ -81,7 +81,7 @@ <h2 class="section-title" translate>HomePage.StepsSection.Title</h2>
                     </div>
                     <div>
                         <p translate>HomePage.StepsSection.Step4.Text</p>
-                        <a routerLink="/participation" class="button button_medium button_secondary" translate>
+                        <a href="/participation" class="button button_medium button_secondary" translate>
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html]: It should be `routerLink` instead of `href`, shouldn't it?

Comment (grigol-brd) [reply ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html]: nah my bad i'll correct it

Comment (grigol-brd) [reply ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html]: it's done

File: ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html [hunk 2]
```diff
@@ -13,7 +13,7 @@ <h1 class="no-margin" translate>
                         class="button button_medium button_primary" translate>
                         Common.Buttons.CountMeIn
                     </a>
-                    <a routerLink="more-details" class="button button_medium button_secondary" translate>
+                    <a routerLink="/participation" class="button button_medium button_secondary" translate>
```
Comment (GiCharkviani) [inline ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html:16]: **Optional:**
I suggest the following syntax: `{{'id' | translate}}`
As it offers acceptable readability.

Comment (GiCharkviani) [reply ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html:16]: You can check [here](https://www.codeandweb.com/babeledit/tutorials/how-to-translate-your-angular-app-with-ngx-translate)

**Review by GiCharkviani — APPROVED** [review]

**Pull request merged**